aptos-move/framework/aptos-framework/sources/transaction_context.move

```diff
@@ -46,6 +57,128 @@ module aptos_framework::transaction_context {
         auid.unique_address
     }
 
+    /// Return the sender's address for the current transaction.
+    /// This function aborts if called outside of the transaction prologue, execution, or epilogue phases.
+    native fun sender_internal(): address;
+    public fun sender(): address {
+        assert!(features::transaction_context_extension_enabled(), error::invalid_state(ETRANSACTION_CONTEXT_EXTENSION_NOT_ENABLED));
+        sender_internal()
+    }
+
+    /// Return the list of the secondary signers for the current transaction.
+    /// If the current transaction has no secondary signers, this function returns an empty vector.
+    /// This function aborts if called outside of the transaction prologue, execution, or epilogue phases.
+    native fun secondary_signers_internal(): vector<address>;
+    public fun secondary_signers(): vector<address> {
```

Comment:
Can these be friends to the prologue/epilogue rather than public?

@gedigi Can you take a look at this functions from the security standpoint? Want to make sure that these newly added interfaces won't leak info that would enable transactions to do something funky.

Reply:
@vgao1996 , I will talk to you about this in Slack.